### ISSUE TITLE
A question: about adding multiple dates/times to calendar

### ISSUE THREAD

**Issue by Ycitsci** [SciStarter/Circuit#15]
9. At individual project > add to calendar, can multiple dates/times be added?
![9 adding multiple times ](https://user-images.githubusercontent.com/92533858/137411049-5e9cb9c4-2ec1-44a4-8408-b06e1bc0d584.png)

**Comment by SciStarter** [comment]
Yes, they can. If the event has multiple dates, each date can be added separately.
